sample/src/androidTest/java/com/schibsted/spain/barista/sample/SeekBarTest.java

```diff
@@ -22,20 +25,39 @@
   public FailureHandlerValidatorRule handlerValidator = new FailureHandlerValidatorRule();
 
   @Test
-  public void checkSetProgress() {
+  public void checkSeekBarSetProgress() {
     setProgressTo(R.id.seek_bar, 16);
     assertDisplayed("16");
   }
 
   @Test
-  public void checkSetProgressToMax() {
+  public void checkSeekBarSetProgressToMax() {
     setProgressToMax(R.id.seek_bar);
     assertDisplayed("100");
   }
 
   @Test
-  public void checkSetProgressToMin() {
+  public void checkSeekBarSetProgressToMin() {
     setProgressToMin(R.id.seek_bar);
     assertDisplayed("0");
   }
+
+  @Test
+  public void checkRatingBarSetProgress() {
+    setRatingTo(R.id.rating_bar, 3);
+    assertDisplayed("3.0");
+  }
+
+  @Test
+  public void checkRatingBarSetProgressToMax() {
+    setRatingToMax(R.id.rating_bar);
+    assertDisplayed("7.0");
+  }
+
+  @Test
+  public void checkRatingBarSetProgressToMin() {
+    setRatingTo(R.id.rating_bar, 3);
```

Comment:
This is tricky, i know

But RatingBar (and ProgressBar)  doesn't change if current value is the same as new value, and it doesn't invoke the listener, so view will not change...